examples/src/main/java/io/kubernetes/client/examples/ExecExample.java

```diff
@@ -32,19 +42,15 @@ public static void main(String[] args) throws IOException, ApiException, Interru
         Configuration.setDefaultApiClient(client);
 
         Exec exec = new Exec();
-        final Process proc = exec.exec("default", "nginx-2371676037-czqx3", new String[] {"sh", "-c", "echo foo"}, true);
+        boolean tty = System.console() != null;
```

Comment:
No need to fix in this PR, but I suggest we create the pod too so the example is self contained. here is the python example: https://github.com/kubernetes-incubator/client-python/blob/master/examples/exec.py